cmd/dc4bc_cli/main.go

```diff
@@ -839,6 +838,15 @@ func proposeSignMessageCommand() *cobra.Command {
 	}
 }
 
+func getSignID(rawID string) string {
+	letterBytes := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	tail := make([]byte, 5)
+	for i := range tail {
+		tail[i] = letterBytes[rand.Intn(len(letterBytes))]
```

Comment:
Hm... seems like the function always returns the same string for the same rawID on each execution of the cli command because you don't call rand.Seed(...)

crypto/rand is more secure btw